jira-test/jira-test-docker/src/test/it/org/xwiki/contrib/jira/test/ui/PastePluginIT.java

```diff
@@ -117,9 +140,16 @@ void pasteLinkWithTicket(TestUtils setup)
         copyInClipboard(LINK_JIRA_TICKET, setup);
 
         ViewPage vp = setup.gotoPage(TEST_PAGE_REFERENCE);
-        WikiEditPage wikiEditPage = pasteClipboardInCKEditorAndEditWiki(vp, setup, false);
+        WYSIWYGEditPage editPage = vp.editWYSIWYG();
+        CKEditor ckEditor = new CKEditor("content").waitToLoad();
+        setup.getDriver().findElement(By.className("cke_notification_close")).click();
+        String paste = Keys.chord(Keys.CONTROL, "v");
+        ckEditor.getRichTextArea().sendKeys(paste);
 
-        String content = wikiEditPage.getContent();
+        setup.getDriver().waitUntilCondition(
+            webDriver -> ckEditor.getRichTextArea().getText().equals("XWIKI-1000 Improve PDF Output"));
```

Comment:
There's also something sketchy here (whenever `setup.getDriver()` is called in a test, it usually means some method lacking in a PO. Sonce `getContent()` is called on the wiki editor, I don't see how waiting will do anything?

Note that I think we need to verify the display when in the WYSIWYG editor, and not in the wiki editor. What we can verify in the wiki editor is the syntax used to insert the macro.

Reply:
>it usually means some method lacking in a PO

Indeed. For these tests I depend on the CKeditor POs that, on version 14.10.20, are very lacking. The POs are improved in the later versions. Maybe i could put a TODO comment to update the tests when upgrading parent?

> once getContent() is called on the wiki editor, I don't see how waiting will do anything?

The method is called on the CKEditor and it returns the iframe used by it. Inserting a macro in CKEditor is not instantaneous. As a measure of precaution I'm waiting for the content to change so we dont run into flickering issues. There might be a nicer way to do it on the versions where the POs are more complex.